runner/runners/invoke.go

```diff
@@ -154,7 +155,7 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 	case st = <-processCh:
 	}
 
-	log.Infof("run done. id: %v, status: %+v, cmd: %+v, checkout: %v", id, st, cmd, co.Path())
+	log.Infof("run done. id: %v, status: %+v, checkout: %v", id, st, co.Path())
```

Comment:
I have a utility to split sickle and scheduler logs into lists of successes/failures - if we think it would be useful for worker logs we should change this to match (or at least be closer to scheduler or sickle format:
run done:  <cmd>
Status: COMPLETE
Exit Code: <exit code>
Stdout: <stdout ID/URL>
Stderr: <stderr ID/URL>